src/app/register/supplier/FormComponent.tsx

```diff
@@ -1,24 +1,39 @@
 "use client";
 
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Formik, Form, Field, FormikProps } from "formik";
-import { Input, ButtonPrimary, FormItem } from "@/components/atoms";
+import { Input, ButtonPrimary, FormItem, Select } from "@/components/atoms";
 import * as Yup from "yup";
 import { IRegisterAccount } from "@/types/auth";
 import { useRouter } from 'next/navigation'
 import Auth from "@/service/auth.service";
+import Sup from '@/service/supplier.service';
+
+interface categorieProps {
```

Comment:
```suggestion
interface ICategoriesProps {
```

Reply:
Change this to `types` folder and export interface, because CategoriesProps can be used in different locations